plugins/example_plugin/windows/ExamplePlugin.vcxproj

```diff
@@ -15,19 +15,21 @@
     <ProjectGuid>{95E21B2C-C18A-4CED-8509-585CB2570FDE}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>ExamplePlugin</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>

```

Comment:
This breaks the bots; is it VS 2019+ only?

Also, apparently my comment on the previous PR was ambiguous, I meant for this change and the toolset changes to each be their own separate PRs, not split out into one new combined PR. The SDK change is less straightforward (more so than I thought, apparently), while the PlatformToolset change is trivial and doesn't require discussion.